### PR TITLE
chore: add favicon

### PR DESCRIPTION
**Pull request by SamanthaAO**
# Description
- add favicons-webpack-plugin
- update webpack to use plugin and favicon
- update docusaurus config
- remove unused static favicons for the old brand

Screen Shot 2020-06-25 at 4 45 44 PM

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast-dna/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast-dna/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist

- [x] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: sites/website/docusaurus.config.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ module.exports = {
     tagline: "The adaptive interface system for modern web experiences",
     url: "https://www.fast.design",
     baseUrl: "/",
-    favicon: "img/favicon/favicon.ico",
+    favicon: "bundle/favicon.ico",
```
Comment (awentzel) [inline sites/website/docusaurus.config.js:6]: shouldn't this be pulled from CDN?

Comment (janechu) [reply sites/website/docusaurus.config.js:6]: I've just merged my PR that includes the favicon, so yes we should now have a link to it to the effect of https://static.fast.design/assets/manifest/favicon.ico

File: sites/fast-website/webpack.common.js [hunk 2]
```diff
@@ -4,9 +4,11 @@ const path = require("path");
 const { CleanWebpackPlugin } = require("clean-webpack-plugin");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
 const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const FaviconsWebpackPlugin = require("favicons-webpack-plugin");
```
Comment (janechu) [inline sites/fast-website/webpack.common.js:7]: We should not need this now that the `favicon.ico` file is now on CDN (https://static.fast.design/assets/manifest/favicon.ico)

File: sites/website/docusaurus.config.js (hunk 1 above)
Comment (SamanthaAO) [reply sites/website/docusaurus.config.js]: Thank you for letting me know. I have updated it to the CDN.

File: sites/fast-website/webpack.common.js (hunk 2 above)
Comment (SamanthaAO) [reply sites/fast-website/webpack.common.js]: Thank you for the comment. I have removed the FaviconsWebpackPlugin and added a link to the CDN in the index.html.

**Pull request merged**